pkgs/development/python-modules/mapbox-earcut/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, setuptools
+, numpy
+, pybind11
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mapbox-earcut";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "skogler";
+    repo = "mapbox_earcut_python";
+    rev = "257c2c82819acf72b2b757f200c160e55b9427fc";
+    sha256 = "14z2vj13a1qcaqc6jvg00f367la5470bclvfi9mmlvpac3hhhywd";
+    };
+
+    nativeBuildInputs = [ setuptools ];
```

Comment:
Is that required at runtime?

Reply:
No. But it is required at buildtime. So I thought putting in into `nativeBuildInputs` wouldn't be a bad idea. Isn't `nativeBuildInputs` meant for specifying buildtime deps? I know i don't need to specify this because `setuptoolsBuildPhase` will automatically use `setuptools`.